examples/dynamic-bridge-app/linux/include/Attribute.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include <app/ConcreteAttributePath.h>
+
+#include "DataModel.h"
+
+// This is the interface to the attribute implementation
```

Comment:
"...that permits retrieval of metadata about the attribute as well as set/get the underlying value"